config/v0.3.0/resource.go

```diff
@@ -47,16 +47,20 @@ func keyVal(k string, v any) attribute.KeyValue {
 }
 
 func newResource(res *Resource) *resource.Resource {
-	if res == nil || res.Attributes == nil {
+	if res == nil ||
+		res.SchemaUrl == nil && res.Attributes == nil && res.AttributesList == nil && res.Detectors == nil {
```

Comment:
I think if the user provides an empty (null) resource we should honor this configuration.